src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -6717,7 +6717,6 @@ address generate_avx_ghash_processBlocks() {
     StubRoutines::_fence_entry                = generate_orderaccess_fence();
 
     // platform dependent
-    StubRoutines::x86::_get_previous_fp_entry = generate_get_previous_fp();
```

Comment:
The body of `generate_get_previous_fp()` can also be removed.

Reply:
Thanks for noticing.  I meant to delete it.